scripts/test_runner.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+ENV_PRODUCT="${ENV_PRODUCT}"
+ENV_TFVARS="${ENV_TFVARS}"
+TESTDIR="${TESTDIR}"
```

Comment:
Again I vote we expand all multi-word variables 
TEST_DIR="${TEST_DIR}"
 and so on